components/filter/test/filter.visual-diff.js

```diff
@@ -194,6 +194,27 @@ describe('d2l-filter', () => {
 					await visualDiff.screenshotAndCompare(page, this.test.fullTitle(), { captureBeyondViewport: false });
 				});
 			});
+			[
+				'single-set-empty',
+				'single-set-single-selection',
+				'single-set-single-selection-select-all',
+				'single-set-multi-selection',
+				'single-set-multi-selection-no-search',
+				'single-set-multi-selection-no-search-select-all',
+				'single-set-multi-selection-all-selected',
+				'multiple-empty',
+				'multiple-selected'
+			].forEach(type => {
+				it(type, async function() {
+					const selector = `#${type}`;
+					await openWithKeyboard(page, selector);
+					page.waitForTimeout(50);
+					await page.$eval (selector, async(elem)  => {
+						requestAnimationFrame(async() => await elem.updateComplete);
+					});
+					await visualDiff.screenshotAndCompare(page, `${this.test.fullTitle()}-keyboard-focused`, { captureBeyondViewport: false });
```

Comment:
Are these testing anything particular interesting about keyboard focus in filter beyond what dropdown itself is testing?

What I mean by that is if we have good dropdown + keyboard + focus tests, then I don't think we need to re-test them for everything that uses dropdown unless it's a special/custom case.

@svanherk FYI

Reply:
Trying to get caught up on the context of this PR - what will the change to filter focus be?

I was gonna check the visual-diffs to see but I think it's very unhappy because `${this.test.fullTitle()}-keyboard-focused` isn't an id in the html files, and the test names are duplicated.  I think you actually want to wrap the above test in another describe like this:
```
describe('keyboard-focused', () => {
		[
				'single-set-empty',
				'single-set-single-selection',
				'single-set-single-selection-select-all',
				'single-set-multi-selection',
				'single-set-multi-selection-no-search',
				'single-set-multi-selection-no-search-select-all',
				'single-set-multi-selection-all-selected',
				'multiple-empty',
				'multiple-selected'
			].forEach(type => {
				it(type, async function() {
					const selector = `#${type}`;
					await openWithKeyboard(page, selector);
					page.waitForTimeout(50);
					await page.$eval (selector, async(elem)  => {
						requestAnimationFrame(async() => await elem.updateComplete);
					});
					await visualDiff.screenshotAndCompare(page, `${this.test.fullTitle()}`, { captureBeyondViewport: false });
				});
		});
});
```

Reply:
Hey @svanherk no specific changes regarding filter but there are some focus changes where opening dropdowns using a mouse will now cause the focus to move to within the menu (but with no styles). Seemed like filter hit a bunch of focus use cases since it has focus trap + a bunch of internal components. I can remove the tests if its preferred since they are technically testing other functionality, but I though having some snapshots of what keyboard focus vs mouse focus is could be useful.